### DISCUSSION TITLE
ユーザをミュートやブロックするときに理由を書ける機能が欲しい

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#522] [要望・アイデア・不具合]
興味がない投稿数が連続しているので一時的にミュートしたり、コメントが嫌われたので刺激しないようにミュートしたり、スパムだと思うからブロックしたり、理由はいろいろあると思います。
一時的にミュートしたユーザをほとぼりが冷めたころに解除しようと思ってもどのユーザか忘れてしまっています。
どのような理由でミュート/ブロックしたかコメントを残せる機能があると嬉しいです。

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご意見ありがとうございます！

直近の開発優先度などから考えると、現時点では対応の予定はありませんが
他にも同様のご意見をいただく場合は検討したいと思います🙏
